### ISSUE TITLE
Invalid contract address for `$GSWIFT` token

### ISSUE THREAD

**Issue by tkowalczyk** [OffchainLabs/arbitrum-token-lists#120]
Dear OffchainLabs Team,

I noticed that on UI part of your bridge after using `Arbed CMC List` there is a wrong contract address for `$GSWIFT` token which on Arbitrum is:
https://arbiscan.io/token/0x580e933d90091b9ce380740e3a4a39c67eb85b4c
and on Ethereum also:
https://etherscan.io/token/0x580e933d90091b9ce380740e3a4a39c67eb85b4c

This bridge behaviour leads to receiving wrong token on Arbitrum:
https://arbiscan.io/tx/0x054c39e7187b2756473cfa3729de8eddcbc119f582a693808a77801308a9d2b9

1. What steps should be taken to allow users bridging `$GSWIFT` token correctly?
2. How to change this contract address on `Arbed CMC List`?

Screen:
![obraz](https://github.com/OffchainLabs/arbitrum-token-lists/assets/1682564/3d2ca114-ee49-4275-863e-bd28c0ba398a)

**Comment by chrstph-dvx** [comment]
**tl;dr: funds are not lost, user can withdraw to Mainnet**

Unfortunately, the token is not a token bridged from Ethereum which uses the generic token gateway. We are going to limit the token to withdraw only (Arbitrum One to Ethereum) to prevent further errors. The user can withdraw the token (0x88e5369f73312eba739dcdf83bdb8bad3d08f4c8) and receive GSWIFT (0x580e933d90091b9ce380740e3a4a39c67eb85b4c) on Ethereum. They can deposit through the GSWIFT’s official bridge

**Comment by tkowalczyk** [comment]
Hi @chrstph-dvx great to hear that, is there any way tu support correct token on official bridge? with whom should I contact with to add `$GSWIFT` token to one of these available lists?

**Comment by fionnachan** [comment]
Hi @tkowalczyk, our official bridge only supports bridging through our standard gateways and does not integrate with others' smart contracts for bridging, so the best course of actions for users is to use GSWIFT's official bridge to perform bridging. We have updated our bridge website to disable deposits of GSWIFT, but still allow the withdrawal of `0x88e5369f73312eba739dcdf83bdb8bad3d08f4c8` from Arbitrum One to Ethereum.
